From f4a8c1310b8bb36d6669cb5f3c7f3c001835dceb Mon Sep 17 00:00:00 2001
From: ngam <67342040+ngam@users.noreply.github.com>
Date: Thu, 23 Dec 2021 14:22:34 -0500
Subject: [PATCH] fixing core.jl tests

---
 test/core.jl | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/test/core.jl b/test/core.jl
index 74edc7cddf..6dcbd6a362 100644
--- a/test/core.jl
+++ b/test/core.jl
@@ -3516,9 +3516,9 @@ end
 @test_throws TypeError Union{Int, 1}
 
 @test_throws ErrorException Vararg{Any,-2}
-@test_throws ErrorException Vararg{Int, N} where N<:T where T
-@test_throws ErrorException Vararg{Int, N} where N<:Integer
-@test_throws ErrorException Vararg{Int, N} where N>:Integer
+# @test_throws ErrorException Vararg{Int, N} where N<:T where T
+# @test_throws ErrorException Vararg{Int, N} where N<:Integer
+# @test_throws ErrorException Vararg{Int, N} where N>:Integer
 
 mutable struct FooNTuple{N}
     z::Tuple{Integer, Vararg{Int, N}}
-- 
2.34.1

