From c0df114623ea6b1a7a9b57eafbf5e76683a58728 Mon Sep 17 00:00:00 2001
From: ngam <67342040+ngam@users.noreply.github.com>
Date: Sun, 26 Dec 2021 21:11:45 -0500
Subject: [PATCH] correctly point in @projectname --project

---
 test/cmdlineargs.jl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/cmdlineargs.jl b/test/cmdlineargs.jl
index 7f075e231e..75a8a53ee1 100644
--- a/test/cmdlineargs.jl
+++ b/test/cmdlineargs.jl
@@ -112,7 +112,7 @@ let exename = `$(Base.julia_cmd()) --startup-file=no`
     if !Sys.iswindows()
         expanded = abspath(expanduser("~/foo"))
         @test occursin(expanded, readchomp(`$exename --project='~/foo' -E 'Base.active_project()'`))
-        @test occursin(expanded, readchomp(setenv(`$exename -E 'Base.active_project()'`, "JULIA_PROJECT" => "~/foo", "HOME" => homedir())))
+        @test occursin(expanded, readchomp(setenv(`$exename -E 'Base.active_project()'`, "JULIA_PROJECT" => "~/foo", "HOME" => homedir(), "JULIA_DEPOT_PATH" => ENV["JULIA_DEPOT_PATH"])))
     end
 
     # --quiet, --banner
-- 
2.32.0 (Apple Git-132)

