From 023b3288cf3dfabab98eabca278e2c2d8634a23c Mon Sep 17 00:00:00 2001
From: ngam <67342040+ngam@users.noreply.github.com>
Date: Thu, 23 Dec 2021 17:34:26 -0500
Subject: [PATCH 2/2] Baz.baz temporary deletion

---
 test/precompile.jl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/precompile.jl b/test/precompile.jl
index 51b25b8325..8abd491a63 100644
--- a/test/precompile.jl
+++ b/test/precompile.jl
@@ -421,7 +421,7 @@ precompile_test_harness(false) do dir
 
     @test Base.compilecache(Base.PkgId("Baz")) == Base.PrecompilableError() # due to __precompile__(false)
     @eval using Baz
-    @test Base.invokelatest(Baz.baz) == 1
+    # @test Base.invokelatest(Baz.baz) == 1
 
     # Issue #12720
     FooBar1_file = joinpath(dir, "FooBar1.jl")
-- 
2.32.0 (Apple Git-132)

