From 67f620177ee9f321cd62b9e86114439761542dcb Mon Sep 17 00:00:00 2001
From: ngam <ngam@users.noreply.github.com>
Date: Mon, 27 Dec 2021 01:37:48 +0000
Subject: [PATCH 3/4] correctly point in @projectname --project

---
 test/cmdlineargs.jl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/cmdlineargs.jl b/test/cmdlineargs.jl
index fb206acf03..3d1efdd0d2 100644
--- a/test/cmdlineargs.jl
+++ b/test/cmdlineargs.jl
@@ -112,7 +112,7 @@ let exename = `$(Base.julia_cmd()) --startup-file=no --color=no`
     if !Sys.iswindows()
         let expanded = abspath(expanduser("~/foo/Project.toml"))
             @test expanded == readchomp(`$exename --project='~/foo' -e 'println(Base.active_project())'`)
-            @test expanded == readchomp(setenv(`$exename -e 'println(Base.active_project())'`, "JULIA_PROJECT" => "~/foo", "HOME" => homedir()))
+            @test expanded == readchomp(setenv(`$exename -e 'println(Base.active_project())'`, "JULIA_PROJECT" => "@foo", "HOME" => homedir(), "JULIA_DEPOT_PATH" => ENV["JULIA_DEPOT_PATH"]))
         end
     end
 
-- 
2.34.1

