From e4f0e7c7f750dc0f27fd59c1fe7cd8d70dbd2fc9 Mon Sep 17 00:00:00 2001
From: ngam <67342040+ngam@users.noreply.github.com>
Date: Sun, 26 Dec 2021 21:45:50 -0500
Subject: [PATCH 4/4] another occurence of pointing

---
 test/cmdlineargs.jl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/cmdlineargs.jl b/test/cmdlineargs.jl
index 3d1efdd0d2..7fb7e5ad9e 100644
--- a/test/cmdlineargs.jl
+++ b/test/cmdlineargs.jl
@@ -119,7 +119,7 @@ let exename = `$(Base.julia_cmd()) --startup-file=no --color=no`
     # handling of @projectname in --project and JULIA_PROJECT
     let expanded = abspath(Base.load_path_expand("@foo"))
         @test expanded == readchomp(`$exename --project='@foo' -e 'println(Base.active_project())'`)
-        @test expanded == readchomp(setenv(`$exename -e 'println(Base.active_project())'`, "JULIA_PROJECT" => "@foo", "HOME" => homedir()))
+        @test expanded == readchomp(setenv(`$exename -e 'println(Base.active_project())'`, "JULIA_PROJECT" => "@foo", "HOME" => homedir(), "JULIA_DEPOT_PATH" => ENV["JULIA_DEPOT_PATH"]))
     end
 
     # --quiet, --banner
-- 
2.34.1

