From 31e9de0137db669f1a41e933da701b0e2c7ea6f2 Mon Sep 17 00:00:00 2001
From: ngam <ngam@users.noreply.github.com>
Date: Mon, 27 Dec 2021 01:36:51 +0000
Subject: [PATCH 2/3] disable testing Baz.baz == 1

---
 test/precompile.jl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/precompile.jl b/test/precompile.jl
index 51b25b8325..8abd491a63 100644
--- a/test/precompile.jl
+++ b/test/precompile.jl
@@ -470,7 +470,7 @@ precompile_test_harness(false) do dir
 
     @test Base.compilecache(Base.PkgId("Baz")) == Base.PrecompilableError() # due to __precompile__(false)
     @eval using Baz
-    @test Base.invokelatest(Baz.baz) == 1
+    # @test Base.invokelatest(Baz.baz) == 1
 
     # Issue #12720
     FooBar1_file = joinpath(dir, "FooBar1.jl")
-- 
2.34.1

