From d3284af6e4326be51fbb3b6a3625d146928d3c2b Mon Sep 17 00:00:00 2001
From: Mark Kittisopikul <markkitt@gmail.com>
Date: Mon, 19 Sep 2022 13:12:22 -0400
Subject: [PATCH] Defer codesigning to conda-forge

---
 contrib/codesign.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/contrib/codesign.sh b/contrib/codesign.sh
index 03866c4bb1..1155ff9c18 100755
--- a/contrib/codesign.sh
+++ b/contrib/codesign.sh
@@ -8,6 +8,9 @@ usage() {
     exit 0
 }
 
+# Disable redundant codesigning in conda-forge, which uses a custom binary
+exit 0
+
 # Default codesign identity to `-` if not provided
 if [ -z "${1}" ]; then
     MACOS_CODESIGN_IDENTITY="-"
-- 
2.34.1

