From da3ced7c171b3fabf8e3ffa836912dfc47b67c85 Mon Sep 17 00:00:00 2001
From: Mark Kittisopikul <kittisopikulm@janelia.hhmi.org>
Date: Fri, 2 Sep 2022 05:32:22 -0400
Subject: [PATCH] Debug precompile

---
 contrib/generate_precompile.jl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/generate_precompile.jl b/contrib/generate_precompile.jl
index a10d195229..c246a24d7e 100644
--- a/contrib/generate_precompile.jl
+++ b/contrib/generate_precompile.jl
@@ -242,7 +242,7 @@ const HELP_PROMPT = "help?> "
 
 function generate_precompile_statements()
     start_time = time_ns()
-    debug_output = devnull # or stdout
+    debug_output = stdout
     sysimg = Base.unsafe_string(Base.JLOptions().image_file)
 
     # Extract the precompile statements from the precompile file
-- 
2.37.2

