From e50a1c6fb20608e1d909797b554637969aa980c7 Mon Sep 17 00:00:00 2001
From: Mark Kittisopikul <kittisopikulm@janelia.hhmi.org>
Date: Fri, 2 Sep 2022 20:45:50 -0400
Subject: [PATCH] Force use of build env curl

---
 deps/tools/jldownload | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/deps/tools/jldownload b/deps/tools/jldownload
index 99822195fe..7a32f499c6 100755
--- a/deps/tools/jldownload
+++ b/deps/tools/jldownload
@@ -6,7 +6,8 @@
 CACHE_HOST=https://cache.julialang.org
 
 WGET=$(which wget 2>/dev/null)
-CURL=$(which curl 2>/dev/null)
+#CURL=$(which curl 2>/dev/null)
+CURL=$CONDA_PREFIX/bin/curl
 FETCH=$(which fetch 2>/dev/null)
 
 TIMEOUT=15 # seconds
-- 
2.32.1 (Apple Git-133)

