--- a/test/cmdlineargs.jl	2025-10-17 22:04:28.000000000 +0100
+++ b/test/cmdlineargs.jl	2025-10-31 12:52:06.721421398 +0000
@@ -715,39 +715,6 @@
 
     # -g
     @test readchomp(`$exename -E "Base.JLOptions().debug_level" -g`) == "2"
-    # --print-before/--print-after with pass names is broken on Windows due to no-gnu-unique issues
-    if !Sys.iswindows()
-        withenv("JULIA_LLVM_ARGS" => "--print-before=BeforeOptimization") do
-            let code = readchomperrors(`$exename -g0 -E "@eval Int64(1)+Int64(1)"`)
-                @test code[1]
-                code = code[3]
-                @test occursin("llvm.module.flags", code)
-                @test !occursin("llvm.dbg.cu", code)
-                @test !occursin("int.jl", code)
-                @test !occursin("name: \"Int64\"", code)
-            end
-            let code = readchomperrors(`$exename -g1 -E "@eval Int64(1)+Int64(1)"`)
-                @test code[1]
-                code = code[3]
-                @test occursin("llvm.module.flags", code)
-                @test occursin("llvm.dbg.cu", code)
-                # TODO: consider moving test to llvmpasses as this fails on some platforms
-                # without clear reason
-                @test_skip occursin("int.jl", code)
-                @test !occursin("name: \"Int64\"", code)
-            end
-            let code = readchomperrors(`$exename -g2 -E "@eval Int64(1)+Int64(1)"`)
-                @test code[1]
-                code = code[3]
-                @test occursin("llvm.module.flags", code)
-                @test occursin("llvm.dbg.cu", code)
-                # TODO: consider moving test to llvmpasses as this fails on some platforms
-                # without clear reason
-                @test_skip occursin("int.jl", code)
-                @test occursin("name: \"Int64\"", code)
-            end
-        end
-    end
 
     # --check-bounds
     let JL_OPTIONS_CHECK_BOUNDS_DEFAULT = 0,
@@ -1273,15 +1240,6 @@
 @test readchomp(`$(Base.julia_cmd()) -e 'module Hello; export main; (@main)(args) = println("hello"); end; using .Hello'`) == "hello"
 @test readchomp(`$(Base.julia_cmd()) -e 'module Hello; export main; (@main)(args) = println("hello"); end; import .Hello'`) == ""
 
-# test --bug-report=rr
-if Sys.islinux() && Sys.ARCH in (:i686, :x86_64) # rr is only available on these platforms
-    mktempdir() do temp_trace_dir
-        @test success(pipeline(setenv(`$(Base.julia_cmd()) --bug-report=rr-local -e 'exit()'`,
-                                      "JULIA_RR_RECORD_ARGS" => "-n --nested=ignore",
-                                      "_RR_TRACE_DIR" => temp_trace_dir); #=stderr, stdout=#))
-    end
-end
-
 @testset "--heap-size-hint" begin
     exename = `$(Base.julia_cmd())`
     @test errors_not_signals(`$exename --heap-size-hint -e "exit(0)"`)
