--- test/cmdlineargs.jl.orig	2025-10-16 11:48:49.907366176 +0100
+++ test/cmdlineargs.jl	2025-10-16 11:51:38.229834080 +0100
@@ -671,39 +671,6 @@
 
     # -g
     @test readchomp(`$exename -E "Base.JLOptions().debug_level" -g`) == "2"
-    # --print-before/--print-after with pass names is broken on Windows due to no-gnu-unique issues
-    if !Sys.iswindows()
-        withenv("JULIA_LLVM_ARGS" => "--print-before=BeforeOptimization") do
-            let code = readchomperrors(`$exename -g0 -E "@eval Int64(1)+Int64(1)"`)
-                @test code[1]
-                code = code[3]
-                @test occursin("llvm.module.flags", code)
-                @test !occursin("llvm.dbg.cu", code)
-                @test !occursin("int.jl", code)
-                @test !occursin("name: \"Int64\"", code)
-            end
-            let code = readchomperrors(`$exename -g1 -E "@eval Int64(1)+Int64(1)"`)
-                @test code[1]
-                code = code[3]
-                @test occursin("llvm.module.flags", code)
-                @test occursin("llvm.dbg.cu", code)
-                # TODO: consider moving test to llvmpasses as this fails on some platforms
-                # without clear reason
-                @test_skip occursin("int.jl", code)
-                @test !occursin("name: \"Int64\"", code)
-            end
-            let code = readchomperrors(`$exename -g2 -E "@eval Int64(1)+Int64(1)"`)
-                @test code[1]
-                code = code[3]
-                @test occursin("llvm.module.flags", code)
-                @test occursin("llvm.dbg.cu", code)
-                # TODO: consider moving test to llvmpasses as this fails on some platforms
-                # without clear reason
-                @test_skip occursin("int.jl", code)
-                @test occursin("name: \"Int64\"", code)
-            end
-        end
-    end
 
     # --check-bounds
     let JL_OPTIONS_CHECK_BOUNDS_DEFAULT = 0,
