From 96f8f3061dcd50e100454a65d885f36c669c2508 Mon Sep 17 00:00:00 2001
From: ngam <67342040+ngam@users.noreply.github.com>
Date: Sat, 8 Jan 2022 18:05:35 -0500
Subject: [PATCH] upgrade downloads temp

---
 stdlib/Downloads.version | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/stdlib/Downloads.version b/stdlib/Downloads.version
index 056fc38d82..92b057c3b6 100644
--- a/stdlib/Downloads.version
+++ b/stdlib/Downloads.version
@@ -1,4 +1,4 @@
-DOWNLOADS_BRANCH = release-1.5
-DOWNLOADS_SHA1 = 26d79afcde9cf837a331fce023b31f1d3699700c
+DOWNLOADS_BRANCH = release-1.6
+DOWNLOADS_SHA1 = 43899e71eed709866f96dd624f1c369a1c8e26a3
 DOWNLOADS_GIT_URL := git://github.com/JuliaLang/Downloads.jl.git
 DOWNLOADS_TAR_URL = https://api.github.com/repos/JuliaLang/Downloads.jl/tarball/$1
-- 
2.34.1


