diff --git a/stdlib/SuiteSparse_jll/src/SuiteSparse_jll.jl b/stdlib/SuiteSparse_jll/src/SuiteSparse_jll.jl
--- stdlib/SuiteSparse_jll/src/SuiteSparse_jll.jl
+++ stdlib/SuiteSparse_jll/src/SuiteSparse_jll.jl
@@ -53,33 +53,33 @@
     const libspqr = "libspqr.dll"
     const libsuitesparseconfig = "libsuitesparseconfig.dll"
     const libumfpack = "libumfpack.dll"
 elseif Sys.isapple()
    const libamd = "@rpath/libamd.3.dylib"
    const libbtf = "@rpath/libbtf.2.dylib"
    const libcamd = "@rpath/libcamd.3.dylib"
    const libccolamd = "@rpath/libccolamd.3.dylib"
-    const libcholmod = "@rpath/libcholmod.4.dylib"
+    const libcholmod = "@rpath/libcholmod.5.dylib"
    const libcolamd = "@rpath/libcolamd.3.dylib"
    const libklu = "@rpath/libklu.2.dylib"
    const libldl = "@rpath/libldl.3.dylib"
    const librbio = "@rpath/librbio.4.dylib"
    const libspqr = "@rpath/libspqr.4.dylib"
    const libsuitesparseconfig = "@rpath/libsuitesparseconfig.7.dylib"
    const libumfpack = "@rpath/libumfpack.6.dylib"
 else
    const libamd = "libamd.so.3"
    const libbtf = "libbtf.so.2"
    const libcamd = "libcamd.so.3"
    const libccolamd = "libccolamd.so.3"
-    const libcholmod = "libcholmod.so.4"
+    const libcholmod = "libcholmod.so.5"
    const libcolamd = "libcolamd.so.3"
    const libklu = "libklu.so.2"
    const libldl = "libldl.so.3"
    const librbio = "librbio.so.4"
    const libspqr = "libspqr.so.4"
    const libsuitesparseconfig = "libsuitesparseconfig.so.7"
    const libumfpack = "libumfpack.so.6"
 end
 
 function __init__()
     if Base.USE_GPL_LIBS
